src/src.pro

```diff
@@ -18,8 +18,8 @@
 #    59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             #
 ############################################################################
 
-lessThan(QT_MAJOR_VERSION, 5) {
-  error("requires Qt 5.0 or later")
+lessThan(QT_MAJOR_VERSION, 5)|lessThan(QT_MAJOR_VERSION, 6):lessThan(QT_MINOR_VERSION, 6) {
```

Comment:
Thanks for checking! Questionable behaviour from ``qmake`` for sure. Fixed - have another look!